pyrealm/pmodel/quantum_yield.py

```diff
@@ -321,11 +323,14 @@ def _calculate_kphio(self) -> None:
             ExperimentalFeatureWarning,
         )
 
+        aridity_index = getattr(self.env, "aridity_index")
+        mean_growth_temperature = getattr(self.env, "mean_growth_temperature")
+
         # Calculate enzyme kinetics
         a_ent, b_ent, Hd_base, Ha = self.env.pmodel_const.sandoval_kinetics
         # Calculate activation entropy as a linear function of
         # mean growth temperature, J/mol/K
-        deltaS = a_ent + b_ent * self.env.mean_growth_temperature
+        deltaS = a_ent + b_ent * mean_growth_temperature
```

Comment:
You are not usually using camel case - might be more consistent to use snake case (`delta_s`)